README.md

```diff
@@ -258,6 +258,11 @@ Hydra({
             -- show the hydras name (or "HYDRA:" if not given a name), at the
             -- beginning of an auto-generated hint
             show_name = true,
+            
+            -- if set to true, this will prevent the hydra's hint window from displaying
+            -- immediately.
+            -- Note: you can still show the window manually by calling Hydra:show()
```

Comment:
I think this should be `Hydra.hint:show()`, and maybe we should mention `Hydra.hind:close()` as well.

Reply:
I agree. Done